lib/freedom_patches/sprockets_patches.rb

```diff
@@ -49,3 +49,13 @@ def process_source(source)
     end
   end,
 )
+
+# Skip digest path for workbox assets. They are already in a folder with a digest in the name.
+Sprockets::Asset.prepend(
+  Module.new do
+    def digest_path
+      return logical_path if logical_path.match?(%r{^workbox-.*/})
+      super
+    end
+  end,
+)
```

Comment:
Wonder if we make a note of this in `workbox-tree-builder.js` ?

Reply:
Good call - added :+1: